### PR TITLE
Division on smaller pull request

### PR DESCRIPTION
**Pull request by oklymchuk**

### CODE REVIEW

File: model/password_test.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+func Test_createSalt(t *testing.T) {
+	salt, err := createSalt()
+	assert.NoError(t, err)
+	assert.NotNil(t, salt)
+
+}
+
+func TestEncodePassword(t *testing.T) {
```
Comment (i3odja) [inline model/password_test.go:41]:
```
--- FAIL: TestEncodePassword (0.39s)
    password_test.go:53: 
                Error Trace:    password_test.go:53
                Error:          Not equal: 
                                expected: "$argon2id$v=19$m=65536,t=3,p=1$L/cXOPSeeE9f68JKienFug$t7/BEg7jx2Gjx/OwSFyWgw"
                                actual  : "$argon2id$v=19$m=65536,t=3,p=1$L/cXOPSeeE9f68JKienFug$52t2o11NiXF/gr0wCF514g"
                                
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -$argon2id$v=19$m=65536,t=3,p=1$L/cXOPSeeE9f68JKienFug$t7/BEg7jx2Gjx/OwSFyWgw
                                +$argon2id$v=19$m=65536,t=3,p=1$L/cXOPSeeE9f68JKienFug$52t2o11NiXF/gr0wCF514g
                Test:           TestEncodePassword
FAIL

```

File: model/password_test.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+	saltNew, err := createSalt()
+	assert.NoError(t, err)
+
+	bSalt := base64.RawStdEncoding.EncodeToString(saltNew)
+	hash1 := argon2.IDKey(passByte, saltNew, c.time, c.memory, c.threads, c.keyLen)
+	b64Hash1 := base64.RawStdEncoding.EncodeToString(hash1)
+	wrong := fmt.Sprintf(hashFormat, argon2.Version, c.memory, c.time, c.threads, bSalt, b64Hash1)
+	assert.NotEqual(t, correctPassword, wrong)
+	fmt.Println(wrong)
```
Comment (i3odja) [inline model/password_test.go:63]: It is unnecessary `fmt.Println(wrong)`

File: model/pguser.go [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+	var usr User
+	var salted bool
+	err := ur.db.QueryRow(querySelectInfo, login).Scan(&usr.ID, &usr.Username, &usr.Password,
+		&usr.Email, &usr.FirstName, &usr.LastName, &usr.Phone, &salted)
+	if err != nil {
+		if err == sql.ErrNoRows {
+			return nil, errors.Wrap(err, msgUserDidNotExist)
+		} else {
+			return nil, err
+		}
```
Comment (i3odja) [inline model/pguser.go:95]: What do you think change
```go
else {
    return nil, err
}
```
to 
```go
return nil, err
```
?

File: model/pguser_test.go [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+	userRepo := NewUsersRepo(db)
+
+	assert.Equal(t, db, userRepo.db)
+}
+
+func TestAdd(t *testing.T) {
+	db, mock, err := sqlmock.New()
+	if err != nil {
+		t.Fatalf("an error '%s' was not expected when opening a stub database connection", err)
+	}
```
Comment (i3odja) [inline model/pguser_test.go:33]: It is possible to change it, using `require.NoError()`

File: model/password_test.go (hunk 1 above)
Comment (oklymchuk) [reply model/password_test.go:41]: fixed

File: model/password_test.go (hunk 2 above)
Comment (oklymchuk) [reply model/password_test.go]: Sorry, forget to delete.Fixed

File: model/pguser.go (hunk 3 above)
Comment (oklymchuk) [reply model/pguser.go:93]: You are right. Fixed

File: model/pguser_test.go (hunk 4 above)
Comment (oklymchuk) [reply model/pguser_test.go:33]: No. Because it sqlmock database, if it is not created, we cann't continue test

**Pull request merged**